pareto/utilities/results.py

```diff
@@ -967,7 +1033,7 @@ def plot_bars(input_data, args):
         )
 
         # Update animation settings
-        fig.layout.updatemenus[0].buttons[0].args[1]["frame"]["duration"] = 200
+        fig.layout.updatemenus[0].buttons[0].args[1]["frame"]["duration"] = 800
```

Comment:
When I run the tests locally, they run and complete within 23min. When the tests run on github they do not finish in 35 minutes. I'm not really sure why ...does changing these figure layout affect build time ?

Reply:
@lbianchi-lbl @dangunter can you check this issue? it could be the iplot() method.